### ISSUE TITLE
LOSS of the model

### ISSUE THREAD

**Issue by kaiyangqu** [google-research/seanet#2]
Hi, here is a PAPER REPRODUCING problem. When I use the L1 loss as it was said in the paper, the model turns out to be unsteady and the result seems frustrating even in training set. Is there any additional optimizations which is not mentioned in the paper?

**Comment by amirpashamobinitehrani** [comment]
@kaiyangqu Hi,

Thanks for bringing this up. Have you found the solution to this by any chance?

**Comment by amirpashamobinitehrani** [comment]
@kaiyangqu Hi again, I truly appreciate your response! This might be too much to ask, but do you see any possibility for me to look into the whole implementation? I have been working on the paper for over three months now and nearly close to obtain the full implementation. Due to my lack of confidence and knowledge, I am not certain wether the implementation is correct or not.
